### PR TITLE
Upversion to 4.18

### PR DESCRIPTION
**Pull request by sudomakeinstall2**
# Background / Context

# Issue / Requirement / Reason for change

# Solution / Feature Overview

# Implementation Details

# Other Information

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,8 +25,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a
 
 #####################################################################################################
 # Build the operator image
-# note: update origin-cli-artifacts from `latest` to an appropriate OCP verison during release e.g `4.17`
-FROM quay.io/openshift/origin-cli-artifacts:latest AS origincli
+FROM quay.io/openshift/origin-cli-artifacts:4.17 AS origincli
```
Comment (jc-rh) [inline Dockerfile:28]: this is for the 4.17 branch

**Review by jc-rh — CHANGES_REQUESTED** [review]

**Comment by jc-rh** [comment]
also fix this file in both branches https://github.com/openshift-kni/lifecycle-agent/blob/main/config/default/manager_auth_proxy_patch.yaml

File: Dockerfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a
 
 #####################################################################################################
 # Build the operator image
-# note: update origin-cli-artifacts from `latest` to an appropriate OCP verison during release e.g `4.17`
+# note: update origin-cli-artifacts from `latest` to an appropriate OCP verison during release e.g `4.18`
```
Comment (jc-rh) [inline Dockerfile:28]: New release version is typically not available right away.  So it needs to remain latest

File: config/default/manager_auth_proxy_patch.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
       - name: kube-rbac-proxy
-        image: quay.io/openshift/origin-kube-rbac-proxy:4.14
+        image: quay.io/openshift/origin-kube-rbac-proxy:4.18
```
Comment (jc-rh) [inline config/default/manager_auth_proxy_patch.yaml:17]: Wonder if this image is available

Comment (sudomakeinstall2) [reply config/default/manager_auth_proxy_patch.yaml:17]: i pulled it

**Review by jc-rh — APPROVED** [review]

**Pull request merged**